src/t.ts

```diff
@@ -19,14 +8,14 @@ export const getSnapshot = (target: any) => {
 export const t = {
   model,
   string,
-  Date,
+  Date: date,
   boolean,
-  finite: null,
-  float: null,
-  identifier: null,
-  identifierNumber: null,
-  integer: null,
-  null: null,
+  finite: "to be implemented",
+  float: "to be implemented",
+  identifier: "to be implemented",
+  identifierNumber: "to be implemented",
+  integer: "to be implemented",
+  null: "to be implemented",
```

Comment:
I also made these changes to be a little clearer about the intentionally unimplemented types.